seqr/management/commands/reload_saved_variant_json.py

```diff
@@ -31,20 +33,28 @@ def handle(self, *args, **options):
             logger.info("Project: " + project.name)
 
             saved_variants = SavedVariant.objects.filter(project=project, family__isnull=False).select_related('family')
+            if options['reset']:
+                for saved_variant in saved_variants:
+                    saved_variant.saved_variant_json = None
+                    saved_variant.save()
```

Comment:
I also made this save variants immediately after retrieving them from elasticsearch - so that if a tunnel to ES times-out in the middle of a large project (eg. myoseq), most variants can still be updated.
Then, with this option if you first clear all saved_variant_json values, you can easily see which ones have been updated.

Reply:
So a concern I have is that the way we currently load variants, we fetch a whole list of them and then add some additional annotations to everything in the list. While thats more relevant for mongo, it does still happen for a few fields for the es variants as well. Are you sure that the point at which its saving these is after those annotations have been added?

Reply:
good point, but yes, variant_utils._retrieve_saved_variants_json calls add_extra_info_to_variants_project.